src/main/java/com/github/black0nion/blackonionbot/commands/fun/ConnectFourCommand.java

```diff
@@ -2,6 +2,7 @@
 
```

Comment:
If you can be bbothered to you can use the SlashCommandEvent methods instead of manually sending everything, made that back when the new method didn't exist yet